### ISSUE TITLE
URL citations are not found

### ISSUE THREAD

**Issue by ultralapse** [UMLCloudComputing/rowdyLLM#10]
URL citations are no longer found in the responses. This can easily be fixed by writing a function to extract the URL key from the metadata dictionary. If that doesn't work, we would need to fetch the metadata file straight from the S3 bucket.

**Comment by noahshayne** [comment]
I just tested it utilizing the rowdyLLM and the discord bot, and both gave links once requested. Do you mean that even if unrequested, the bot should provide citations?

**Comment by ultralapse** [comment]
Yup. This PR should fix the issue https://github.com/UMLCloudComputing/rowdyLLM/pull/13

**Comment by ultralapse** [comment]
URLs show up on locally run instances but are not showing up on official instance at nightlyrowdy.streamlit.app. Currently looking into this.

**Comment by ultralapse** [comment]
Issue has been fixed
